src/helix/impl/props.cljc

```diff
@@ -72,6 +72,24 @@
 (defn merge-obj [o1 o2]
   #?(:cljs (js/Object.assign o1 o2)))
 
+(defn seq-to-class [class]
```

Comment:
Just realized this won't work as expected - if the class is a quoted vector like `'[foo bar]`, in the maro expand time its value would be `(quote '[foo bar])`.  

Looks like I need more macro magic to handle this, or move this to the cljs land and convert it at runtime.

Reply:
addressed in https://github.com/lucywang000/helix/commit/aa0bee8